extensions/wikia/AdEngine/js/slot/adSlot.js

```diff
@@ -56,9 +56,40 @@ define('ext.wikia.adEngine.slot.adSlot', [
 		return iframe;
 	}
 
+	function getRecoveredProviderContainer(providerContainer) {
+		var elementId = providerContainer.childNodes.length > 0 && providerContainer.childNodes[0].id,
+			recoveredElementId = win._sp_.getElementId(elementId),
+			element = doc.getElementById(recoveredElementId);
+
+		log(['getRecoveredProviderContainer for container', providerContainer], log.levels.debug, logGroup);
+		log(['getRecoveredProviderContainer recovered element ID', recoveredElementId], log.levels.debug, logGroup);
+		log(['getRecoveredProviderContainer recovered element', element], log.levels.debug, logGroup);
+
+		if (element && element.parentNode) {
+			log(['getRecoveredProviderContainer recovered provider container', element.parentNode], log.levels.debug, logGroup);
+			return element.parentNode;
+		}
+
+		return null;
```

Comment:
we use result of this function in two places: `SlotTweaker.js` and `uapVideo.js` and in any of those we don't check for presence of this element. I propose remove this line and add checks there.